tools/osbuilder/rootfs-builder/nvidia/nvidia_chroot.sh

```diff
@@ -42,7 +42,7 @@ set_driver_version_type() {
 	elif [[ "$nvidia_gpu_stack" =~ version=([^,]+) ]]; then
 		driver_version="${BASH_REMATCH[1]}"
 	else
-		echo "No known driver spec found. Please specify \"latest\", \"lts\", or \"driver=<VERSION>\"."
+		echo "No known driver spec found. Please specify \"latest\", \"lts\", or \"version=<VERSION>\"."
```

Comment:
Can you move this change into the first commit please?

Reply:
Ah shoot.